### ISSUE TITLE
Could you please offer me the model and the code to train the model?

### ISSUE THREAD

**Issue by w20ss11** [asimshankar/java-tensorflow#1]
I have tried to add code in model.py. But an error occured at this line "
sess.runner().feed("save/Const", checkpointPrefix).addTarget("save/restore_all").run();    "

Here is my code:

import tensorflow as tf
import os
import numpy as np
x = tf.placeholder(tf.float32, shape=[None, 1, 1], name='input')
y = tf.placeholder(tf.float32, shape=[None, 1, 1], name='target')

y_ = tf.identity(tf.layers.dense(x, 1), name='output')

loss = tf.reduce_mean(tf.square(y_ - y), name='loss')
optimizer = tf.train.GradientDescentOptimizer(learning_rate=0.01)
train_op = optimizer.minimize(loss, name='train')

saver = tf.train.Saver(tf.global_variables())
init = tf.global_variables_initializer()
sess = tf.Session()
sess.run(init)

saver_def = tf.train.Saver().as_saver_def()
with open('graph.pb', 'w') as f:
    f.write(tf.get_default_graph().as_graph_def().SerializeToString())

images = np.array([5,7,3,2,1,6,4,8,9,0],dtype=np.float32)
labels = np.array([17,23,11,8,5,20,14,26,29,2],dtype=np.float32)
for step in range(10):
    sess.run(train_op,feed_dict={x:images[step].reshape([1,1,1]), y:labels[step].reshape([1,1,1])})
    if step == 9:
        checkpoint_path = os.path.join("./model1")
        saver.save(sess, checkpoint_path, global_step=step)
